release/src/main/scripts/download_github_actions_artifacts.py

```diff
@@ -30,7 +30,7 @@
 import requests
 
 GH_API_URL_WORKLOW_FMT = "https://api.github.com/repos/{repo_url}/actions/workflows/build_wheels.yml"
-GH_API_URL_WORKFLOW_RUNS_FMT = "https://api.github.com/repos/{repo_url}/actions/workflows/{workflow_id}/runs"
+GH_API_URL_WORKFLOW_RUNS_FMT = "https://api.github.com/repos/{repo_url}/actions/workflows/{workflow_id}/runs?branch={ref}"
```

Comment:
is this supposed to be branch ref name or tag ref?

Reply:
the intent seems to be ref of the commit corresponding to the tag. Is 'branch' in the url just a way how GH actions refer to actions performed on a commit id?

Reply:
Oh, sorry I missed this question. Yes. GHA just uses that name. I confirmed it works for a tag. The usual git terminology for such a thing is a `ref` so I went with that.